Origami/src/components/App.js

```diff
@@ -120,160 +98,48 @@ class App extends React.Component {
   }
 
   setSessionFlag(access_token, username) {
-    localStorage.setItem("access_token", access_token);
-    localStorage.setItem("username", username);
-    localStorage.setItem("gh_access_token_time", Date.now());
+    localStorage.setItem('access_token', access_token);
+    localStorage.setItem('username', username);
+    localStorage.setItem('gh_access_token_time', Date.now());
   }
 
   initiateLogin() {
-    window.location = "/auth/github/login/";
+    window.location = '/auth/github/login/';
   }
 
   logout() {
     this.props.loginactions.Logout();
     this.clearSessionFlag();
-    window.location = "/";
+    window.location = '/';
   }
 
   getDocs() {
     window.location = ORIGAMI_READ_THE_DOCS;
   }
 
-  handleClickAfterLogin(e, navLinks) {
```

Comment:
This was for the navbar right? What have we replaced it with?

Reply:
We are using Default navbar of coreUI. @AvaisP